components/mpas-seaice/docs/tech-guide/index.md

```diff
@@ -0,0 +1,99 @@
+
+### Radiation
+
+The Delta-Eddington radiation scheme of Briegleb & Light (2007) has been updated to the Dang et al. (2019) SNICAR-AD model, to ensure radiative consistency across all snow surfaces in E3SM, including on land, ice sheets and sea ice. The SNICAR-AD radiative transfer code includes five-band snow single-scattering properties, two-stream Delta-Eddington approximation with the adding–doubling technique, and parameterization for correcting the near-infrared (NIR) snow albedo biases when solar zenith angle exceeds 75 degrees (Dang et al., 2019).
+
+### Snow
+
+A new snow-on-sea-ice morphology has been added to E3SMv2 that includes the effects of wind redistribution: losses to leads and meltponds, and the piling of snow against ridges. Snow grain radius, now a prognosed tracer field on sea ice, evolves according to temperature gradient and wet snow metamorphism and feeds back to the SNICAR-AD radiative model up to a dry maximum of 2800 μm. Fresh snow falls at a grain radius of 54.5 μm, and five vertical snow layers replace the previous single snow layer atop each of the five sea ice thickness categories retained from E3SMv1.
+
+A paper describing the advanced snow physics is in preparation.
+
+### Biogeochemistry
+
+This section is under construction, pending the full merge of BGC codes in Icepack and the older column physics package.
+
+## Coupling of MPAS-seaice within E3SM
+
```

Comment:
Suggest adding the following to point potential users to the place in the code that lists all variables involved in MPAS-SI coupling:

Several files within MPAS-SI control the coupling of fields between MPAS-SI and other E3SM components via the coupler. A list of all variables being passed to and from MPAS-SI can be found in **components/mpas-seaice/driver/mpassi_cpl_indices.F** 
Variables named with the prefix **_x2i_** are passed from the coupler to MPAS-SI, variables with the prefix **_i2x_** are passed from MPAS-SI to the coupler. 
For example, all coupled variables are named with the following convention: 
"index_i2x_Si_ithick" refers to a state variable of ice (Si) that gets passed from MPAS-SI to the coupler (i2x) for the ice thickness (ithick). "index_x2i_So_t" refers to an ocean state variable (So) passed from the the coupler to MPAS-SI (x2i) for ocean temperature (t).